models/project/src/mc-project/applied-data-fence/generator.ts

```diff
@@ -0,0 +1,14 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import { TAppliedDataFence } from './types';
+
+const generator = Generator<TAppliedDataFence>({
+  fields: {
+    type: fake(() => 'Store'),
+    // TODO: use more realistic values for action rights
```

Comment:
I'm not sure about how these values looks like.
@tdeekens maybe you can give as a hint?

Reply:
Sad truth is that we only have one type of data fence. It's the store. Back then we thought we'd eventually have more ways to data fence the data in a project.

Reply:
Thanks Tobi.

I'm sorry but I meant about the group prop.

Perhaps these are application names?
We could update this the same way we did for `applied-action-right` generator.